### ISSUE TITLE
zsysctl rollback functionality?

### ISSUE THREAD

**Issue by Dave--G** [ubuntu/zsys#228]
This is more a question and perhaps a request for added functionality.

In the use-case of a failed system update (etc), zsys looks great for booting the system to a previous zfs snapshot to get running again.  However, when the computer is subsequently rebooted, it again tries to boot to the failed update.

How is the user to make permanent rolling back the system to a previous state?  Am I missing something?  I don't see any provision for a `zsysctl rollback [state]` function.  Ideally zsysctl rollback could detect the current booted system state and provide a simplified path to recovering the system to this state.

In the absence of a rollback function, could someone please provide official documentation for how to properly accomplish this?  It seems to be the missing piece of the puzzle that would complete zsys and make it an outstanding tool for managing ubuntu zfs based systems.

(Or has @canonical really dropping zfs support?  Zfs certainly seems to be be neglected recently which is sad to see after so much great work was put into supporting it.)

**Comment by didrocks** [comment]
zsys is supposed to commit on any successful rollback. A successful rollback is defined by reaching default.target (see the unit on https://github.com/ubuntu/zsys/blob/master/systemd/zsys-commit.service). It could be something that fails on your systemd, preventing this unit to run?

**Comment by Dave--G** [comment]
Thank you for responding.  I will check into it.  And I think I need to read the material (I just now discovered) that you provided here https://discourse.ubuntu.com/t/zfs-focus-on-ubuntu-20-04-lts-blog-posts/16355 more thoroughly before commenting further.  I'll close this bug soon if I have no further issues or questions.

And thank you for providing such an excellent piece of software, @didrocks, regardless of this specific issue frankly I'm impressed with it's design and a little disappointed it (and zfs) hasn't received greater recognition.  

The reason I mentioned loss of zfs support is I noticed zsys wasn't included in the 22.04 distro.  I had to install it manually.  Hopefully this was a simple omission and not intentional.
